### ISSUE TITLE
Android Auto delayed start button, both screens 'usable' but not usable and improvements to voice instructions

### ISSUE THREAD

**Issue by 0xshuzo** [organicmaps/organicmaps#7006]
**Describe the issue**
A few bugs and improvements for android auto.

- ~~If some destination is picked (either bookmark or searched) sometimes the screen will render the computed route but will fall back to the layout to pick if you want to search or pick bookmarks, etc. After a few minutes it then suddenly shows the start button to start the route.~~ Updated by @AndrewShkrob. Should be fixed already
- ~~If you switch fast between android auto and your device screen you will see the app showing up on both screens without the notice that it is running on the other device. Both screens won't also work again. Sometimes both screens will also turn black which seems related to #6716 in some way.~~ Updated by @AndrewShkrob. Should work better now

- An improvement would be to let the voice instructions be played as an instruction sound in the car (Google Maps does that). Using this all other currently playing car media is reduced in volume and more importantly, in most cars it is possible to set a different volume for instructions and media. Unfortunately, I couldn't find any documentation for that till now but if I come across anything I'll let you know. 

**Steps to reproduce**
Described above. If more details are needed, let me know.

**Expected behaviour**
1. Start button should be rendered as soon as the route is computed and shown on the screen.
2. Either, the app should only be usable on one device, not both.

**System information:**
 - Operating system and its version: [Android 14]
 - Organic Maps version: [2023.12.25-6-Google]
 - Device Model: [Samsung Galaxy S21]

**Additional context**
Logs:

[Delayed start button](https://github.com/organicmaps/organicmaps/files/13790101/logcat_delayed_start.log)

[Both screens visible](https://github.com/organicmaps/organicmaps/files/13790102/logcat_blackscreen.log)

**Comment by AndrewShkrob** [comment]
>An improvement would be to let the voice instructions be played as an instruction sound in the car (Google Maps does that). Using this all other currently playing car media is reduced in volume and more importantly, in most cars it is possible to set a different volume for instructions and media. Unfortunately, I couldn't find any documentation for that till now but if I come across anything I'll let you know.

Could you please elaborate on this? Are you saying that, currently, when you use Android Auto, the instructions sound is played through your mobile phone? That's weird.

**Comment by niels-beier** [comment]
No, it's played on the car. Some cars differentiate between media sound and voice instruction sound (for example Volkswagen, BMW, Ford, etc.). Therefore you can adjust the volume of those seperately. Since Android Auto supports to differentiate between those sound types as well, it would be nice to have this implemented for the voice instructions of OM as well because till now you're only able to have the voice instructions at the same level as the media playback, which in my case is too loud for the media but exactly right for the voice instructions of OM.

**Comment by AndrewShkrob** [comment]
Thank you for the clarification. Could you please check if other navigation apps support this feature? For example, Waze or Maps.Me. This verification is needed to confirm that implementing this feature is indeed possible.

**Comment by niels-beier** [comment]
I'll have a look the next few days.

**Comment by niels-beier** [comment]
So Waze and Yandex Navigator do support this feature, Maps.Me doesn't. It didn't test it yet, but it seems like OsmAnd allows the user to choose which output device is used for [voice prompts](https://osmand.net/docs/user/navigation/guidance/voice-navigation#voice-guidance-output). In osmandapp/OsmAnd#9719 it seems like changing to one of these options (most likely to phone call audio) in their settings kind of does this effect. I'll try to test it for OsmAnd, but it seems like, there are options to address this.

**Comment by geniusled** [comment]
Ive just setup voice on organic maps and found myself noticing this with android auto.
Currently the same volume control is used for music and guidance. 

On google maps, if a guidance voice is heard, pressing the volume at this time allows to control the volume seperately from music.
On my headunit (oem subaru) when adjusting music volume it shows a music note icon while when adjusting navigation voice it shows a speaker icon, so they are seen as seperate sources

**Comment by biodranik** [comment]
@0xshuzo please check if it works for you in the latest Organic Maps update.

**Comment by AndrewShkrob** [comment]
Users from other similar issues confirmed that sound is now fixed. I'm closing the issue
